command_line/integrate.py

```diff
@@ -463,15 +467,13 @@ def run_integration(params, experiments, reference=None):
     """
             )
         elif unmatched:
-            logger.info("")
-            logger.info("*" * 80)
-            logger.info(
-                "Warning: %d reference spots were not matched to predictions",
-                unmatched.size(),
+            msg = (
+                "Warning: %d reference spots were not matched to predictions"
+                % unmatched.size()
             )
-            logger.info("*" * 80)
-            logger.info("")
-        rubbish.extend(unmatched)
+            border = "\n".join(("", "*" * 80, ""))
+            logger.info("".join((border, msg, border)))
```

Comment:
If this truly is a "Warning", can we change this to
```python
logger.warning("%d reference spots were not matched to predictions", unmatched.size())
```
and get rid of all the border newline sparkly sugar stuff?

Reply:
I wasn't convinced this was actually a warning, and would like to defer to someone who has more knowledge about `dials.integrate` as to whether this is routinely encountered.

Reply:
I _think_ this is info. I see the message a lot